foundationdb/src/main/java/site/ycsb/db/foundationdb/FoundationDBClient.java

```diff
@@ -60,6 +60,8 @@
   private static final String EXTERNAL_CLIENT_DIRECTORY_DEFAULT = ".";
   private static final String TRACE_DIRECTORY            = "foundationdb.tracedirectory";
   private static final String TRACE_DIRECTORY_DEFAULT    = "";
+  private static final String RETRY_LIMIT                = "foundationdb.retrylimit";
+  private static final String RETRY_LIMIT_DEFAULT        = "5";
```

Comment:
This should default to zero (retry forever).  FDB often goes unavailable for minutes at a time during load, which would create holes in the keyspace.  In turn, it's likely that YCSB's verify will notice, and declare the database corrupted.